examples/plotting/README.rst

```diff
@@ -0,0 +1,4 @@
+Making Custom Layouts for Plots
+===============================
```

Comment:
Wondering if this is not too specific?
```suggestion
Data visualization
==================
```
To make the corresponding heading in the user guide?